content/en/blog/_posts/2021-09-06-memory-qos-cgroups-v2/index.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: blog
+title: 'Alpha in Kubernetes v1.22: Quality-of-Service for Memory Resources'
```

Comment:
```suggestion
title: Quality-of-Service for Memory Resources
```
Since the 1.22 release was several months ago, the title can be decoupled from the 1.22 release